tests/Feature/Api/V1/UserRecentAchievementsTest.php

```diff
@@ -44,6 +44,11 @@ public function testGetUserRecentAchievements(): void
         $unlock3Date = $now->clone()->subMinutes(1);
         $this->addSoftcoreUnlock($this->user, $achievement3, $unlock3Date);
 
+        // Refresh the achievements.
+        $achievement1 = Achievement::findOrFail($achievement1->id);
```

Comment:
just $achievement->fresh() should work, too, if i'm not mistaken.

Reply:
Interesting! I didn't know about this. It appears there are two methods to choose from: `fresh()` and `refresh()`.

`fresh()` fetches a new instance and assigns it, eg:
```php
$achievement = $achievement->fresh();
```

`refresh()` is even more minimal. It updates the current instance in-place, eg:
```php
$achievement->refresh();
```

Latest has been updated to use `refresh()`.